packages/tables/examples/selectable-table.md

```diff
@@ -66,22 +74,48 @@ const isSelectAllChecked = (selectedRows, rows) => {
     </HeaderRow>
   </Head>
   <Body>
-    {state.rows.map(row => (
-      <Row key={row.id} isSelected={state.selectedRows[row.id]}>
+    {state.rows.map((row, index) => (
+      <Row key={row.id} isSelected={row.selected}>
         <Cell isMinimum>
           <Field>
             <Checkbox
-              checked={state.selectedRows[row.id] ? true : false}
+              checked={row.selected ? true : false}
```

Comment:
Since `row.selected` is a boolean, could this line be simplified by removing the ternary operation?

```jsx
checked={row.selected}
```

Reply:
Updated.